RCAIDE/Library/Plots/Thermal_Management/plot_cross_flow_heat_exchanger_conditions.py

```diff
@@ -53,21 +53,25 @@ def plot_cross_flow_heat_exchanger_conditions(cross_flow_hex, results, coolant_l
     # get line colors for plots 
     line_colors   = cm.inferno(np.linspace(0,0.9,len(results.segments)))     
 
-    fig = plt.figure(save_filename)
-    fig.set_size_inches(width,height) 
-    axis_0 = plt.subplot(1,1,1)
-    axis_1 = plt.subplot(3,2,1)
-    axis_2 = plt.subplot(3,2,2) 
-    axis_3 = plt.subplot(3,2,3) 
-    axis_4 = plt.subplot(3,2,4)
-    axis_5 = plt.subplot(3,2,5) 
-    axis_6 = plt.subplot(3,2,6)         
-    b_i = 0 
-
-   
-    axis_0.plot(np.zeros(2),np.nan*np.zeros(2), color = line_colors[0], marker = ps.markers[b_i], linewidth = ps.line_width,label= cross_flow_hex.tag) 
-    axis_0.grid(False)
-    axis_0.axis('off')  
+    fig_1 = plt.figure(save_filename + '_Effectiveness' )
+    fig_2 = plt.figure(save_filename + '_Inlet_Air_Temp' )
+    fig_3 = plt.figure(save_filename + '_Coolant_Flowrate' )
+    fig_4 = plt.figure(save_filename + '_Air_Flowrate' )
+    fig_5 = plt.figure(save_filename + '_Power' )
+    fig_6 = plt.figure(save_filename + '_Inlet_Air_Pressure' )
+    fig_1.set_size_inches(width,height) 
+    fig_2.set_size_inches(width,height) 
+    fig_3.set_size_inches(width,height) 
+    fig_4.set_size_inches(width,height) 
+    fig_5.set_size_inches(width,height) 
+    fig_6.set_size_inches(width,height) 
+    axis_1 = fig_1.add_subplot(1,1,1)
+    axis_2 = fig_2.add_subplot(1,1,1)
+    axis_3 = fig_3.add_subplot(1,1,1) 
+    axis_4 = fig_4.add_subplot(1,1,1) 
+    axis_5 = fig_5.add_subplot(1,1,1)
+    axis_6 = fig_6.add_subplot(1,1,1)        
+    b_i = 0  
```

Comment:
Two Many figures will be created for multiple heat exchanger if introduced.... it is not advisable to do this...